doc/configuration.plist

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+  <key>com.google.Chrome.extensions.cfoidkdkbbpjjlaidafdgkeeglomoaij</key>
```

Comment:
Let's use the production ID (`pkehgijcmpdhfbdbbnkijodmdjhbjlgp`) as we do in the other samples.